platforms/documentation/docs/src/docs/userguide/authoring-builds/structuring/sharing_build_logic_between_subprojects.adoc

```diff
@@ -349,17 +349,17 @@ An improper way to share build logic between subprojects is _cross-project confi
 
 TIP:  Avoid using `subprojects {}` and `allprojects {}`.
 
-With cross-configuration, build logic can be injected into a subproject which is not obvious when looking at its build script.
+With <<glossary.adoc#sub:cross_configuration,cross-configuration>>, build logic can be injected into a subproject which is not obvious when looking at its build script.
 
 In the long run, cross-configuration usually grows in complexity and becomes a burden.
-Cross-configuration can also introduce configuration-time coupling between projects, which can prevent optimizations like configuration-on-demand from working properly.
+<<glossary.adoc#sub:cross_configuration,Cross-configuration>> can also introduce configuration-time coupling between projects, which can prevent optimizations like configuration-on-demand from working properly.
 
 === Convention plugins versus cross-configuration
 
-The two most common uses of cross-configuration can be better modeled using convention plugins:
+The two most common uses of <<glossary.adoc#sub:cross_configuration,cross-configuration>> can be better modeled using convention plugins:
 
 1. Applying plugins or other configurations to subprojects of a certain type. +
-Often, the cross-configuration logic is `if subproject is of type X, then configure Y`.
+Often, the <<glossary.adoc#sub:cross_configuration,cross-configuration>> logic is `if subproject is of type X, then configure Y`.
```

Comment:
💭 Maybe only the first mention of cross-configuration in each `===` section should be a link to make the text a little less busy.

Reply:
Fixed.